.ci/lib.sh

```diff
@@ -357,24 +357,24 @@ gen_clean_arch() {
 		delete_containerd_cri_stale_resource
 	fi
 
-	if [[ $CI_JOB != "METRICS" ]]; then
-		#reset k8s service may impact metrics test on x86_64, so limit it to arm64
-		[ $(uname -m) == "aarch64" -a "$(pgrep kubelet)" != "" ] && sudo sh -c 'kubeadm reset -f'
-		info "Remove installed kubernetes packages and configuration"
-		if [ "$ID" == ubuntu ]; then
-			sudo rm -rf /etc/systemd/system/kubelet.service.d
-			sudo apt-get autoremove -y kubeadm kubelet kubectl \
-				$(dpkg -l | awk '{print $2}' | grep -E '^(containerd(.\io)?|docker(\.io|-ce(-cli)?))$')
-		fi
-		# Remove existing k8s related configurations and binaries.
-		sudo sh -c 'rm -rf /opt/cni/bin/*'
-		sudo sh -c 'rm -rf /etc/cni /etc/kubernetes/'
-		sudo sh -c 'rm -rf /var/lib/cni /var/lib/etcd /var/lib/kubelet'
-		sudo sh -c 'rm -rf /run/flannel'
-
-		info "Clean up stale network interface"
-		cleanup_network_interface
+
+	#reset k8s service may impact metrics test on x86_64, so limit it to arm64
+	[ $(uname -m) == "aarch64" -a "$(pgrep kubelet)" != "" ] && sudo sh -c 'kubeadm reset -f'
```

Comment:
What about the other architectures? Would it be better to say `$(uname -m) != "x86_64"`?

Reply:
thanks James

Reply:
@jodh-intel  thanks for the feedback, good point, so I pushed a fix